debugtools/DDR_VM/compile_ddr.xml

```diff
@@ -308,28 +308,12 @@
 		<if>
```

Comment:
Why do we need to keep the os check? Shouldn't `${with-boot-jdk}` work on all platforms?

Reply:
Good point. I'm not even sure why it was there since we only build ddr once on an xlinux box. Likely something leftover from long ago.

Reply:
DDR compilation was only tested/verified on xlinux because of the `extract_structures binary thats used in the job.  I think it might have worked or ran on windows at some point in time.

Reply:
That condition should be removed though, whether this PR or another.